README.md

```diff
@@ -7,61 +7,106 @@
 </a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a>
 
 
-`kind` is a tool for running local Kubernetes clusters using Docker container "nodes".  
-`kind` is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
+`kind` is a tool for running local Kubernetes clusters using Docker container as "nodes".  
+`kind` is primarily designed for testing Kubernetes 1.11+.
+It initially targets the [conformance tests].
 
-If you have [go] and [docker] installed `go get sigs.k8s.io/kind && kind create cluster` is all you need!
+If you have [Go] and [Docker] installed, the following commands are all you need!
 
-<img src="https://gist.githubusercontent.com/BenTheElder/621bc321fc6d9506fd936feb36d32dd0/raw/7fe14e9d0929cab428929ca6c501abc990c07359/kind-create-cluster.gif" alt="2x speed `kind create cluster` demo" />
+```console
+go get -u sigs.k8s.io/kind
+kind create cluster
+export KUBECONFIG=$(kind get kubeconfig-path)
```

Comment:
I'm not sure we should put the whole thing here when we have usage etc below, the paste-able one-liner makes a better impression of how little setup you need to do, while the docs below get into the finer details.

Reply:
That is, the export is not strictly necessary to create a cluster, and you will get the export instructions given to you by the CLI anyhow. I'd rather just add the `-u` to how this was previously.